tests/general/InvalidLinkBearTest.py

```diff
@@ -164,6 +164,14 @@ def test_run(self):
                                                          Queue()),
                                          invalid_file, [1, 1, 1, 1, 1])
 
+    def test_precentage_encoded_url(self):
+        valid_file = """
+        # An url with a precentage-encoded character in path
+        https://img.shields.io/badge/Maintained%3F-yes-green.svg/200
+        """
+
+        self.assertResult(valid_file=valid_file)
```

Comment:
assertResult disappears in https://github.com/coala/coala-bears/pull/1748 , so not helpful to use it now ;-)